### PR TITLE
Remove NSLogs, commented code

### PR DESCRIPTION
**Pull request by gmbharath12**

### CODE REVIEW

File: ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m [hunk 1]
```diff
@@ -82,10 +82,7 @@ - (void)viewDidLoad {
     if ([[NSUserDefaults standardUserDefaults] objectForKey:packagerIPPort] == nil) {
         self->packagerIPandPort = @"localhost:8081";
         [[NSUserDefaults standardUserDefaults] setObject:self->packagerIPandPort forKey:packagerIPPort];
-    }
-    // Store it in NSUserDefaults. default it to YES
-//    [[NSUserDefaults standardUserDefaults] setBool:YES forKey:autoReloadBundle];
-    
+    }
```
Comment (friederbluemle) [inline ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m]: Trailing spaces here. Please remove.

Comment (gmbharath12) [reply ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m]: 👍

Comment (friederbluemle) [reply ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m]: Still there.. ;)


image

Comment (gmbharath12) [reply ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m]: I'm wondering how you are able to see such dotted lines..

Comment (friederbluemle) [reply ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m]: Using this extension:
https://chrome.google.com/webstore/detail/render-whitespace-on-gith/ifdbipohclgnokjgpejhnbjdlgjkkhom?hl=en

Comment (gmbharath12) [reply ern-container-gen-ios/src/hull/ElectrodeContainer/DevAssist/ERNDevSettingsViewController.m]: Sure 👍

**Pull request merged**